.github/workflows/release-tag.yml

```diff
@@ -0,0 +1,47 @@
+name: release-tag
+
+on:
+  repository_dispatch:
+    types: [ metanorma/pngcheck ]
+
+jobs:
+  prepare-ieee:
```

Comment:
```prepare-ieee``` job name looks misleading here. Can we change it ?

Reply:
Right, fixed